300-children-exercises/array-exercises/ex3.rb

```diff
@@ -0,0 +1,17 @@
+=begin
```

Comment:
bài này thử làm 1 vòng lặp thôi xem sao :D

Reply:
```suggestion 
=begin
arr.each_with_index do |a, i|
	if min > a 
		min = a
		count = i
	end
end
puts "Vi tri cac so nho nhat trong mang la #{count+1}"
```cai nay neu gia du co 2 so -10 thi no chi puts ra 1 vi tri cua so -10 thoi

Reply:
arr.each_with_index do |a, i|
	arr.each_with_index do |a, i|
		if min > a 
			min = a	
		end	
	end
	if min == a
		count.push(i + 1)
	end
end
puts "Vi tri cac so nho nhat trong mang la #{count.inspect}"
puts arr.inspect
```suggestion
=begin
```

Reply:
co cach khac nhung cung dung 2 vong lap long vao nhau 
```suggestion
=beginarr.each_with_index do |a, i|
	arr.each_with_index do |a, i|
		if min > a 
			min = a	
		end	
	end
	if min == a
		count.push(i + 1)
	end
end
puts "Vi tri cac so nho nhat trong mang la #{count.inspect}"
puts arr.inspect
```